tools/scripts/refresh-ubuntu-amis.py

```diff
@@ -115,13 +105,15 @@ def update_packer_json(table: List[List[str]], path: str) -> None:
         parser.print_help(sys.stderr)
         sys.exit(1)
 
-    # Do one fetch for all the ubuntu amis.
-    # Note that this is not a public endpoint, but it's the only one that has the gov amis.
-    req = requests.get("https://cloud-images.ubuntu.com/locator/ec2/releasesTable")
+    req = requests.get("https://cloud-images.ubuntu.com/query/focal/server/released.current.txt")
```

Comment:
Since the distribution code name is in two separate URLs, it might be nice if "focal" was a config variable which got f-string'd in.